flaky-tests/ci-providers/semaphore-ci-quickstart.md

```diff
@@ -24,6 +24,25 @@ In your Semaphore dashboard, store your Trunk token in a secret named TRUNK\_TOK
 
 ## Sample Semaphore workflow steps
 
+You can upload test results to Flaky Tests with
+the [`trunk-analytics-uploader`](https://github.com/trunk-io/analytics-uploader) by running it in a stage after
+your tests are complete. There are five different OS/arch builds of the uploader in the latest release.
+Pick the one you need for your testing platform and be sure to download the release on every CI run.
+Do not bake the CLI into a container or VM. This ensures your CI runs are always using the latest build.
```

Comment:
Use a hint/warning box for this for extra attention?

Reply:
I made it bold.